specification/columns/chargecategory.md

```diff
@@ -33,7 +32,6 @@ Allowed values:
 
 | Value      | Description                          |
 | :--------- | :------------------------------------|
-| Adjustment | Any adjustments that are applied after the original usage or purchase row. Adjustments may be related to multiple charges.   |
```

Comment:
I think we need to revert this. We had a convo in TF2 on Mar 6 about keeping Adjustment for rounding errors and any other unanticipated charge types. The description will need to change tho.